Makefile

```diff
@@ -31,23 +31,37 @@ generate: src/curr/generated.rs xdr/curr-version src/next/generated.rs xdr/next-
 
 src/curr/generated.rs: $(sort $(wildcard xdr/curr/*.x))
 	> $@
+ifeq ($(LOCAL_XDRGEN),)
 	docker run -i --rm -v $$PWD:/wd -w /wd docker.io/library/ruby:latest /bin/bash -c '\
 		gem install specific_install -v 0.3.7 && \
 		gem specific_install https://github.com/stellar/xdrgen.git -b $(XDRGEN_VERSION) && \
 		xdrgen --language rust --namespace generated --output src/curr $^ \
 		'
+else
+	docker run -i --rm -v $$PWD/../xdrgen:/xdrgen -v $$PWD:/wd -w /wd docker.io/library/ruby:latest /bin/bash -c '\
+		pushd /xdrgen && bundle install --deployment && rake install && popd && \
```

Comment:
I pushed a commit adding `--deployment` so that the installed gems are installed inside the working directory rather than the system gem store, which will speed up the install process on repeated runs.

Reply:
This won't cause diffs in the xdrgen folder because the files are written to `/vendor/` and that directory is gitignored.